### PR TITLE
Amazon Massive PL Instruction Creator

### PR DESCRIPTION
**Pull request by pawelkiszczak**
I'm glad to share with you 2nd and updated version of the amazon-massive-pl script to create instructions out of the aforementioned dataset.

Please feel free to share all your thoughts and ideas for even more improvements!

@IgorTest19 and @mglabska - I'd be more than happy if you could be the reviewers :)

### CODE REVIEW

File: amazon-massive-pl.py [hunk 1]
```diff
@@ -213,37 +282,131 @@ def load_instructions():
...
-			for worker in item['judgments']:
-				temp['instruction'] = main_dict[type]['instruction']
+			for _ in item['judgments']:
+
+				categories = get_categories_as_string(
+					main_category=main_dict[typ]['output'][item[typ]],
+					categories=main_dict[typ]['categories'])
+
+				'''
+				OPTIONAL DICTIONARY CREATION EXAMPLE
```
Comment (IgorTest19) [inline amazon-massive-pl.py:387]: This is a good example of use, but usually we describe such things in the function/methods description. In this case, it is fine be me to place it there.

**Review by IgorTest19 — APPROVED** [review]
Well described code with good structure and readability.

File: amazon-massive-pl.py [hunk 2]
```diff
@@ -310,22 +324,19 @@ def get_categories_as_string(categories: List,
 
 	if len(categories) > 10 and main_category: 
 		x = 0
+		random_categories = random.choices(list(categories), k=5)
```
Comment (pawelkiszczak) [inline amazon-massive-pl.py]: This change should not happen as it does crash the function. The `random.choices()` shall remain where it was previously.

**Review by pawelkiszczak — COMMENTED** [review]
Suggested changes are accepted, please approve the review.

**Pull request merged**